src/Polyhedron.cuh

```diff
@@ -19,28 +18,64 @@ public:
      */
     __device__ Polyhedron(Face *faces, int n_of_faces);
 
-    /// Forbids copying `Polyhedron` objects
-    __host__ __device__ Polyhedron(const Polyhedron &) = delete;
+    /**
+     * `Polyhedron` object copy assignment operator
+     *
+     * @warning Copying `Polyhedron` object requires copying `Face`s inside it, which can sometimes lead to unexpected
+     *      results (different from when moving!). For details check the documentation of `Face` copy assignment
+     *      operator
+     */
+    __device__ Polyhedron &operator=(const Polyhedron &other);
```

Comment:
Codacy found an issue: ['Polyhedron::operator=' should return 'Polyhedron &'.](https://app.codacy.com/gh/physarumAdv/minds_crawl/pullRequest?prid=5984297)

Reply:
Well, to my mind, it does return what it should. Looks like something is wrong, probably because we are trying to analyze CUDA code as it is C++. I will contact Codacy Support and ask about this